README.md

```diff
@@ -1,6 +1,6 @@
-| CicleCI Build Status | Sonar Code Quality | DockerHub Release | DockerHub Release Candidate | DockerHub Develop |
+| GitHub Actions Build Status | Sonar Code Quality | DockerHub Release | DockerHub Release Candidate | DockerHub Develop |
```

Comment:
As mentioned in my other comment on this PR, let's [1] remove the DockerHub-related columns and [2] create separate tables for the `develop` and `master` branches similar to how the badges are shown on the V2X Hub repository's README.